fonts.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@font-face {
+	font-family: 'Montserrat';
+	src: url('../fonts/Montserrat.woff') format('woff'), url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2');
+	font-weight: normal;
+	font-style: normal;
+}
+@font-face {
+	font-family: 'montserrat_bold';
+	src: url('../fonts/montserrat_bold.woff') format('woff'), url('../fonts/montserrat-v14-latin_cyrillic-700.woff2') format('woff2');
+	font-weight: 700;
+	font-style: normal;
+}
+.newcollection__title { 
+	font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: bold;
+    font-size: 80px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)